lib/rust/ensogl/core/src/animation/loops.rs

```diff
@@ -95,7 +99,8 @@ impl<Callback> RawLoopData<Callback> {
         Self { callback, on_frame, handle_id }
     }
 
-    /// Run the animation frame.
+    /// Run the animation frame. The time fill be converted to [`f32`] because of performance
+    /// reasons. See https://hugotunius.se/2017/12/04/rust-f64-vs-f32.html to learn more.
```

Comment:
I remember we had at some point issues with precision of long-running animations. We should ensure this does not introduce an issue with that again.